File: templates/reserva.py
from flask import Flask, render_template, request

app = Flask(__name__)

@app.route('/reserva', methods=['GET', 'POST'])
def reserva():
    if request.method == 'POST':
        nombre_completo = request.form['nombre_completo']
        fecha_nacimiento = request.form['fecha_nacimiento']
        telefono = request.form['telefono']
        correo = request.form['correo']
        nombre_libro = request.form['nombre_libro']
        fecha_apartado = request.form['fecha_apartado']
        
        print(f"Reserva realizada: {nombre_completo}, {fecha_nacimiento}, {telefono}, {correo}, {nombre_libro}, {fecha_apartado}")
        
        return render_template('reserva.html', mensaje="Reserva realizada con éxito.")
    
    return render_template('reserva.html', mensaje="Por favor, complete el formulario.")

if __name__ == '__main__':
    app.run(debug=True)

    
    return render_template('reserva.html')  

if __name__ == '__main__':
    app.run(debug=True)

File: templates/reserva.php
<?php
$servername = "localhost";
$username = "id22192670_socd1";
$password = "{mO6d^l>YZ2|7rKO";
$dbname = "id22192670_socd";

$conn = new mysqli($servername, $username, $password, $dbname);

if ($conn->connect_error) {
    die("Conexión fallida: " . $conn->connect_error);
}

$nombre_completo = $_POST['nombre_completo'];
$fecha_nacimiento = $_POST['fecha_nacimiento'];
$telefono = $_POST['telefono'];
$correo = $_POST['correo'];
$nombre_libro = $_POST['nombre_libro'];
$fecha_apartado = $_POST['fecha_apartado'];

$sql = "INSERT INTO reservas (nombre_completo, fecha_nacimiento, telefono, correo, nombre_libro, fecha_apartado)
        VALUES ('$nombre_completo', '$fecha_nacimiento', '$telefono', '$correo', '$nombre_libro', '$fecha_apartado')";

if ($conn->query($sql) === TRUE) {
    echo "Reserva realizada con éxito.";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>

